longevity_test.py

```diff
@@ -78,6 +95,21 @@ def run_pre_create_keyspace(self):
         if self.params.get('pre_create_keyspace'):
             self._pre_create_keyspace()
 
+    def _run_validate_large_collections_in_system(self, node):
+        self.log.info("Verifying large collections in system tables on node: {}".format(node))
+        with self.db_cluster.cql_connection_exclusive(node=node) as session:
+            query = "SELECT * from system.large_cells WHERE keyspace_name='large_collection_test'" \
+                    " AND table_name='table_with_large_collection' ALLOW FILTERING"
+            statement = SimpleStatement(query, fetch_size=10)
+            data = list(session.execute(statement))
+            assert len(data) >= 1, "Did not find expected row in system.large_cells"
+
+    def _run_validate_large_collections_warning_in_logs(self, node):
+        self.log.info("Verifying warning for large collections in logs on node: {}".format(node))
+        msg = "Writing large collection"
+        res = list(node.follow_system_log(patterns=[msg], start_from_beginning=True))
+        assert res, f"Did not find expected log message warning: {msg}"
```

Comment:
same.
(Although I'm not sure it's really needed to verify also in logs)

Reply:
This was your request for this test as we have only 2 places where we can check it. The `system.large_cells` table and relevant message in logs.

Reply:
We can keep it as long as it doesn't cause a performance issue for SCT to monitor these messages.